### PR TITLE
Rename handler functions: Update -> Edit

### PR DESCRIPTION
**Pull request by lpichler**
This change makes more consistent naming for function which are handling PATCH methods.

### CODE REVIEW

**Review by MikelAlejoBR — APPROVED** [review]

File: authentication_handlers_test.go [hunk 1]
```diff
@@ -326,7 +326,7 @@ func TestAuthenticationUpdateNotFound(t *testing.T) {
 	c.SetParamValues("not existing uid")
 	c.Request().Header.Add("Content-Type", "application/json;charset=utf-8")
 
-	notFoundAuthenticationUpdate := ErrorHandlingContext(AuthenticationUpdate)
+	notFoundAuthenticationUpdate := ErrorHandlingContext(AuthenticationEdit)
```
Comment (petracihalova) [inline authentication_handlers_test.go]: maybe we can change the names of variable on left side here to `notFoundAuthenticationEdit`
and same on other places in tests

**Review by petracihalova — COMMENTED** [review]
LGTM 👍🏻
I have only tiny comment

File: authentication_handlers_test.go (hunk 1 above)
Comment (lpichler) [reply authentication_handlers_test.go]: that's right - fixed, thanks!

**Review by lindgrenj6 — APPROVED** [review]

File: authentication_handlers_test.go [hunk 2]
```diff
@@ -359,7 +359,7 @@ func TestAuthenticationUpdateBadRequest(t *testing.T) {
 	c.SetParamValues(fixtures.TestAuthenticationData[0].ID)
 	c.Request().Header.Add("Content-Type", "application/json;charset=utf-8")
 
-	badRequestAuthenticationUpdate := ErrorHandlingContext(AuthenticationUpdate)
+	badRequestAuthenticationUpdate := ErrorHandlingContext(AuthenticationEdit)
```
Comment (petracihalova) [inline authentication_handlers_test.go]: I found another place where we can change "update" to "edit"

File: rhc_connection_handlers_test.go [hunk 3]
```diff
@@ -394,7 +394,7 @@ func TestRhcConnectionUpdateNotFound(t *testing.T) {
 	c.SetParamNames("id")
 	c.SetParamValues(invalidId)
 
-	notFoundRhcConnectionUpdate := ErrorHandlingContext(RhcConnectionUpdate)
+	notFoundRhcConnectionUpdate := ErrorHandlingContext(RhcConnectionEdit)
```
Comment (petracihalova) [inline rhc_connection_handlers_test.go]: and here too

File: authentication_handlers_test.go (hunk 2 above)
Comment (lpichler) [reply authentication_handlers_test.go]: done

File: rhc_connection_handlers_test.go (hunk 3 above)
Comment (lpichler) [reply rhc_connection_handlers_test.go]: done

**Comment by lpichler** [comment]
@petracihalova it needed to change also names of test function + 1 struct,

 thanks all for review

**Review by petracihalova — APPROVED** [review]

**Review by lindgrenj6 — APPROVED** [review]

**Review by lindgrenj6 — APPROVED** [review]

**Pull request merged**